.gitignore

```diff
@@ -14,3 +14,4 @@ js/templates.js
 js/ui/*.js
 js/w3c/templates/*
 js/w3c/*.js
+!js/deps/highlight.js
```

Comment:
Is this a must? Can we publish this from its repo instead?

Reply:
We definitely could, but I need to make a request to the w3c system's team, etc. It's not a big deal, but will probably take a few days to set up on the w3c side.

Reply:
Can you live with this for now? I can put thing in motion on the W3C side, but releasing is somewhat broken without patch because the hljs build tool is no more (RIP 💀).

Reply:
I definitely can.